### ISSUE TITLE
Broken icons under custom path

### ISSUE THREAD

**Issue by Haplo164** [jopeek/fvtt-dndbeyond-character-sheet#27]
When foundry is run under a custom path the concentration and ritual icon links are broken.

**Comment by mdemoss** [comment]
I see this problem too. 

The setting in `options.json` :
```json
"routePrefix": "foundry"
```

The un-prefixed requests 404:
```weblog
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
```

The prefixed requests succeed:
```weblog
"GET /foundry/modules/dndbeyond-character-sheet/css/dndbeyond-character-sheet.css HTTP/2.0" 200 4
"GET /foundry/modules/dndbeyond-character-sheet/dndbeyond-character-sheet.js HTTP/2.0" 200 7811 "
"GET /foundry/modules/dndbeyond-character-sheet/images/inspiration.png HTTP/2.0" 200 711 "https:/
"GET /foundry/modules/dndbeyond-character-sheet/images/checkbox.png HTTP/2.0" 200 530 "https://sp
"GET /foundry/modules/dndbeyond-character-sheet/fonts/RobotoCondensed-Regular.woff2 HTTP/2.0" 200
"GET /foundry/modules/dndbeyond-character-sheet/fonts/RobotoCondensed-Bold.woff2 HTTP/2.0" 200 64
"GET /foundry/modules/dndbeyond-character-sheet/css/dndbeyond-character-sheet.css HTTP/2.0" 304 0
"GET /foundry/modules/dndbeyond-character-sheet/dndbeyond-character-sheet.js HTTP/2.0" 304 0 "htt
"GET /foundry/modules/dndbeyond-character-sheet/css/dndbeyond-character-sheet.css HTTP/2.0" 304 0
"GET /foundry/modules/dndbeyond-character-sheet/dndbeyond-character-sheet.js HTTP/2.0" 304 0 "htt
```

**Comment by jopeek** [comment]
Fixed in 1.9.1

**Comment by jopeek** [comment]
I should be able to fix this fairly easily. Give me a few days. On Sep 20, 2022, at 4:38 PM, Matt DeMoss ***@***.***> wrote:﻿
I see this problem too.
The setting in options.json :
"routePrefix": "foundry"
The un-prefixed requests 404:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
"GET /modules/dndbeyond-character-sheet/images/concentration.png HTTP/2.0" 404 3940 
"GET /modules/dndbeyond-character-sheet/images/ritual.png HTTP/2.0" 404 3940 "https:
